### ISSUE TITLE
HP Social Component Layout and Type

### ISSUE THREAD

**Issue by CristianStrit** [TastingKitchen/tasting-kitchen#30]
Please revise to reflect comp in InVision for layout and typography styles (color too) Should have a rollover (hover) state where the image becomes a little opaque (darker) and shows post date (please see InVision). In the new layout, there will be a social icon to show where the post is originally from, title of post, post date and who posted, description, then link out to see more.

<img width="862" alt="Screen Shot 2020-06-30 at 3 26 48 PM" src="https://user-images.githubusercontent.com/67622271/86183422-cb0b9600-bae6-11ea-988c-0e0eef618c02.png">
<img width="719" alt="Screen Shot 2020-06-30 at 3 27 27 PM" src="https://user-images.githubusercontent.com/67622271/86183426-ce9f1d00-bae6-11ea-800d-c2cb51887172.png">

**Comment by erkgmz** [comment]
@CristianStrit I can update the typography and layout discrepancies. Our original MVP did not include social media integration or hover animations. Are these launch blockers?

**Comment by CristianStrit** [comment]
Yes, for now just update the Typography and layout and make them all be blog posts.
